recipe-weather-forecastic/src/recipe_weather_forecastic/pipelines/deploy_forecast/pipeline.py

```diff
@@ -99,7 +99,7 @@ def create_pipeline(**kwargs) -> Pipeline:
                 "registered_model_name": "modified_registered_model_name",
                 "label": "params:deployment.label",
                 "description": "params:deployment.description",
-                "prediction_server_id": "params:credentials.datarobot.prediction_server_id",
+                "default_prediction_server_id": "params:credentials.datarobot.prediction_server_id",
```

Comment:
This should be `prediction_environment_id`

Reply:
I had it like that, but that causes a bug